### ISSUE TITLE
"transform" should be more explicitly described

### ISSUE THREAD

**Issue by TheMangoTrain** [tailwindlabs/tailwindcss.com#502]
Old dev, new to Tailwind. Took me an hour to figure out what was a dead-simple thing.

I have an element that I wanted to become larger on hover...

Naturally I just do the bare minimum:
`<div class="hover:scale-110">Hello</div>`

However I had no clue I had to also have "transform" in there as well:
`<div class="transform hover:scale-110">Hello</div>`

Before this, I went on a 1hr wild goose chase trying to find out how to enable transform as a utility. I see old related posts and I end up messing with corePlugins, etc. 

Ultimately, looking at [someone else's post](https://github.com/tailwindlabs/tailwindcss/issues/1970), I see an answer as `<img class="transform rotate-90">` and I'm throwing my arms up in the air; glad to find a fix but dismayed at why I hadn't seen this sooner.

Going by [TW document for scale](https://tailwindcss.com/docs/scale), there's a statement "Utilities for scaling elements with transform." which is pretty vague. As well, the two popular Tailwind cheatsheets refer to "transform" vaguely as well, and link to TW docs. If searching the TW docs, there isn't actually an entry for transform... or am I wrong?

Whatever the case, it should be more explicitly explained to new users that they need to include "transform" as a class to have any transform work, accompanied by a one liner like the above snippets.

**Comment by alexanderbluhm** [comment]
@adamwathan 
I read qustions on things like this very very often in the Discord as well.
Maybe we could add something like "requires xyz" where xyz is `transform` is this case.
There are a bunch of utilities where beginners think they could use them alone.
For example `flex-col` instead of `flex flex-col` or `grid-cols-2` instead of `grid grid-cols-2`.
It is clear for experienced tailwind users but beginners often struggle.

**Comment by simonswiss** [comment]
Looks like this PR that adds a `transform` documentation page in the "Transforms" section could bring some clarity on that front? 

https://github.com/tailwindlabs/tailwindcss.com/pull/504

**Comment by simonswiss** [comment]
Gonna close this issue now, but this is good feedback, thanks! 

We're going through an internal refresh of the Tailwind docs website in preparation for Tailwind 2.0 - and we'll keep this in mind during the process 👍

**Comment by bc-jasond** [comment]
Looks like the change got reverted in the docs?  I needed to rotate something for the first time and I don't see both classes - only "rotate-X" instead of "transform rotate-X".  A quick search got me what I needed but, seems like it should be in the docs.  Thanks for Tailwind, I'm having a great time with it!

<img width="716" alt="Screen Shot 2022-01-05 at 7 18 34 AM" src="https://user-images.githubusercontent.com/7573868/148242459-f2ce7227-2281-4f83-bd08-c3306dfaca1e.png">
